src/main/java/mate/academy/shop/anotation/AnnotatedClassMap.java

```diff
@@ -1,22 +1,25 @@
 package mate.academy.shop.anotation;
 
+import java.io.FileReader;
+import java.util.HashMap;
+import java.util.Map;
 import mate.academy.shop.dao.BucketDao;
 import mate.academy.shop.dao.ItemDao;
 import mate.academy.shop.dao.OrderDao;
 import mate.academy.shop.dao.UserDao;
+import mate.academy.shop.factory.Factory;
 import mate.academy.shop.service.BucketService;
 import mate.academy.shop.service.ItemService;
 import mate.academy.shop.service.OrderService;
 import mate.academy.shop.service.UserService;
-import mate.academy.shop.factory.Factory;
-
-import java.util.HashMap;
-import java.util.Map;
+import org.apache.log4j.Logger;
 
 public class AnnotatedClassMap {
+    final static Logger logger = Logger.getLogger(FileReader.class);
```

Comment:
the same

Reply:
already pushed changes from the previous task